Prox/Prox/PlaceCarousel/PlaceCarouselCollectionViewCell.swift

```diff
@@ -180,8 +193,40 @@ class PlaceCarouselCollectionViewCell: UICollectionViewCell {
     override func layoutSubviews() {
         super.layoutSubviews()
 
-        NSLayoutConstraint.activate([roundedBackgroundView.heightAnchor.constraint(equalToConstant: contentView.frame.size.height),
-                                     roundedBackgroundView.widthAnchor.constraint(equalToConstant: contentView.frame.size.width)])
+        var updatedConstraints = [roundedBackgroundView.heightAnchor.constraint(equalToConstant: contentView.frame.size.height),
+                                  roundedBackgroundView.widthAnchor.constraint(equalToConstant: contentView.frame.size.width)]
+
+        var deactivateConstraints = [NSLayoutConstraint]()
```

Comment:
Can you explain what's going on here? These more complex constraint combinations are confusing!